pkg/dataloader/prowloader/bigqueryjobs.go

```diff
@@ -52,6 +52,7 @@ func (pl *ProwLoader) fetchProwJobsFromOpenShiftBigQuery() ([]prow.ProwJob, []er
 		"FROM `ci_analysis_us.jobs` " +
 		`WHERE TIMESTAMP(prowjob_completion) > @queryFrom
 	       AND prowjob_url IS NOT NULL
+	       AND prowjob_start IS NOT NULL
```

Comment:
Technically we shouldn't need this any longer correct?

Reply:
correct, since we're now accounting for it later.